Cargo.toml

```diff
@@ -17,6 +17,18 @@ resolver = "2"
 authors = [ "Moondance Labs" ]
 repository = "https://github.com/moondance-labs/tanssi"
 
+[workspace.lints.clippy]
+complexity = { level = "deny", priority = 1 }
+correctness = { level = "deny", priority = 1 }
+erasing_op = { level = "allow", priority = 2 }
+identity_op = { level = "allow", priority = 2 }
+suspicious = { level = "deny", priority = 1 }
+too-many-arguments = { level = "allow", priority = 2 }
```

Comment:
Oh, that's how you deal with overriding parts of a group!

Reply:
Yes, the priority is used as the order in which the flags are passed to the compiler. Previously when using `RUSTFLAGS` we would add an exception for the `too_many_arguments` lint like this:

```
-D clippy::complexity -A clippy::too_many_arguments
```

Because this wouldn't work:

```
-A clippy::too_many_arguments -D clippy::complexity
```